src/com/jwetherell/algorithms/numbers/Integers.java

```diff
@@ -9,9 +9,15 @@ public class Integers {
     private static final BigDecimal ZERO = new BigDecimal(0);
     private static final BigDecimal TWO = new BigDecimal(2);
 
+    private static final int BILLION = 1000000000;
+    private static final int MILLION = 1000000;
+    private static final int THOUSAND = 1000;
+    private static final int HUNDRED = 100;
+    private static final int TEN = 10;
+
     public static final String toBinaryUsingDivideAndModulus(int numberToConvert) {
         int integer = numberToConvert;
-        if (integer<0) throw new IllegalArgumentException("Method argument cannot be negative. number="+integer);
```

Comment:
Rest of code is formatting issues that bothered me when trying to implement the constants.

Reply:
As well as implementing constants obviously